Godel.Microservice.CurrencyRates/MappingHelper/MappingProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+using Godel.Microservice.CurrencyRates.Business.Dto;
+using Godel.Microservice.CurrencyRates.Data.Entities;
+using Godel.Microservice.CurrencyRates.Model;
+
+namespace Godel.Microservice.CurrencyRates.MappingHelper
+{
+    public class MappingProfile : Profile
```

Comment:
it is better to use per model profile and call it like CurrencyProfile, UserProfile etc

Reply:
was renamed